petri-server-core/pom.xml

```diff
@@ -2,13 +2,19 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
     <parent>
         <artifactId>wix-petri</artifactId>
-        <groupId>com.wixpress.common</groupId>
-        <version>1.19.0-SNAPSHOT</version>
+        <groupId>com.wix</groupId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
     </parent>
 
+    <artifactId>petri-server-core</artifactId>
+    <name>Laboratory Spring Integration</name>
```

Comment:
This shouldn't have Spring in it's name (since it doesn't use spring).  and it's also not an integration. 
probably "Laboratory Core " is the most appropriate